escape_test.go

```diff
@@ -0,0 +1,61 @@
+package gocui
```

Comment:
I think we don't want any tests in this library at this point in time.  
Maybe in the further but for now this file is not needed an can be removed.

Reply:
ok, done